ro-base.obo

```diff
@@ -5924,6 +5836,40 @@ property_value: IAO:0000112 "anticoagulant-containing test tube contains measure
 domain: OBI:0000967
 range: COB:0000121
 
+[Typedef]
+id: RO:0017006
+name: has maximum capacity
+def: "A relation that relates a container to a measurement datum that specifies the maximum capacity of the container. Capacity can refer to either weight or volume." []
+property_value: http://purl.org/dc/terms/contributor https://orcid.org/0000-0002-2999-0103
+property_value: http://purl.org/dc/terms/contributor https://orcid.org/0000-0002-5714-991X
+property_value: IAO:0000112 "anticoagulant-containing test tube has maximum capacity 10 ml." xsd:string
+domain: OBI:0000967
+range: COB:0000121
+
+[Typedef]
+id: RO:0017008
+name: owns
+def: "A primitive relation that holds between entities x and y in which y is at x’s full disposal." []
```

Comment:
Looks to me like there is a bad char in here `x’s` instead of `x's` but no worries

Reply:
done